site/outline/schedule.rst

```diff
@@ -0,0 +1,84 @@
+===============
+Course Schedule
+===============
+
+Online courses allow for some flexibility in how class time is integrated into your routine. Nevertheless, there is a
+well-structured and demanding workflow to which you are expected to adhere.
+
+This course is designed for a typical 12 week semester, but is being offered as an accelerated 8 week course. Students
+must be prepared for the pacing of the course.
+
+
+
+.. list-table:: Topic and Lab Schedule
+    :widths: 50 50 50 50
+    :header-rows: 1
+
+    * - Week Number
+      - Monday Date
+      - Topics
+      - Labs
```

Comment:
Are there already links on the page to these topics/labs? May be worthwhile to still link them here for that extra sexy USABILITY

Reply:
Since most of the entries on the page are ranges (e.g. 1 -- 4), it's tough to link nicely without spelling out each one. 

In other words, worse is better.